MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -155,6 +157,7 @@ internal MixedRealityExtensionApp(string globalAppId, MonoBehaviour ownerScript,
 			_assetLoader = new AssetLoader(ownerScript, this);
 			_userManager = new UserManager(this);
 			_actorManager = new ActorManager(this);
+			_physicsBridge = new PhysicsBridge();
```

Comment:
Might we need here "this" for the physics bridge ?

Reply:
There was not need for it yet. We may add it later.